data/build.gradle.kts

```diff
@@ -106,6 +106,7 @@ class ApiKeyProperties(pathToProperties: String, project: Project) {
 
 dependencies {
     implementation(project(mapOf("path" to ":domain")))
+    implementation(project(mapOf("path" to ":compose")))
```

Comment:
@MMurrayAtBR This line should be removed as `data` should not be dependent on `compose`. What code lives in `compose` that is needed in `data`?

cc: @BottleRocket-Colin

Reply:
Hmm, I don't recall adding that. It's also in :app build.gradle, but the git blame annotation is Br* ?
Honestly have no idea why that it's there. Could I have triggered some kind of dependecy adding function in AS?

Reply:
@MMurrayAtBR It should be present in the `:app` module (representing the entire Android application) since the app module contains the Activities and relies on the :compose module to provide the composables for rendering.

The :data module should not be aware of UI and therefore shouldn't need to also depend on :compose, since it shouldn't be using or relying on any code from that module.

Not sure how it got added but please remove just here :)